### ISSUE TITLE
Add virtualenv + zc.buildout based installation to install docs

### ISSUE THREAD

**Issue by do3cc** [plone/documentation#499]

**Comment by svx** [comment]
@do3cc what do you mean ? We base the docs on the UI-Installer and thus use virtualenv

**Comment by do3cc** [comment]
I've heard there is an UI-Installer.
I miss this installation instruction:

```
yum install python-virtualenv
virtualenv .
source bin/activate
echo ...> buildout.cfg
pip install zc.buildout
bin/buildout
```

**Comment by do3cc** [comment]
For context: https://community.plone.org/t/version-conflict-comes-in-plone-as-installing-all-parts-of-buildout-cfg/1450

**Comment by svx** [comment]
@do3cc we should be really carefully with that, I even would put it under something like experienced/advanved docs or under development, do not know that yet, anyway we should think really carefully about that.

We base the docs always base on the UI, we have some tiny part with an example mini-buildout under http://docs.plone.org/manage/installing/requirements.html, which is maybe not the best place :).

Usually and please correct me if I am completely wrong here, I expect from people how serious start developing that they read/know about buildout and version pinning.

Please do not get me wrong here, I am not saying that this would not be good to have in the docs, but I would like to suggest to be careful :)

**Comment by svx** [comment]
@do3cc virtualenv is included in the UI, that is also written in the docs from the UI

**Comment by do3cc** [comment]
I get your point.
The thing is, I know nobody who uses the UI Installer, if somebody needs help like the guy in the link I mentioned, I can't help him debug it.
Also, where should they learn about buildout? I believe the best practices are only transferred by our training, presentations, mouth to mouth and pain

**Comment by svx** [comment]
@do3cc 

You saw that: http://docs.plone.org/manage/installing/requirements.html?highlight=minimal#minimal-build

right ?

Maybe we update that if necessary and move it to another place ?

**Comment by stevepiercy** [comment]
This was addressed in #1746. Closing.
